licenses/cla-individual.md

```diff
@@ -415,3 +415,5 @@ Kamal Habash, @Kamal-Habash, 2020/08/28
 Florian Kohrt, @fkohrt, 2020/09/10
 
 Gerald Liu, @gera2ld, 2020/09/25
+
+Ryan Kramer, @default-kramer, 2020/10/24
```

Comment:
Hi, I think this should be a separated PR, since it can be merged without discussion about the saver PR.